### ISSUE TITLE
How to train a new model with custom dataset

### ISSUE THREAD

**Issue by rose-jinyang** [Tramac/Fast-SCNN-pytorch#35]
Hello
How  are you?
Thanks for contributing this project.
I am going to train a new model with my custom dataset(supervisely-person-dataset).
How should I prepare the training data from my custom dataset?
Thanks

**Comment by Tramac** [comment]
Hi, there are some [dataloader](https://github.com/Tramac/awesome-semantic-segmentation-pytorch/tree/master/core/data/dataloader) may be useful.

**Comment by rose-jinyang** [comment]
Thanks

**Comment by i-am-nut** [comment]
@rose-jinyang did you manage to train with your custom dataset?

**Comment by RazvanPasca** [comment]
Also related to this, do you apply any other preprocessing to the loaded images besides imagenet normalization?

**Comment by i-am-nut** [comment]
hey @RazvanPasca did you manage to train with your custom dataset?
